### ISSUE TITLE
remove extra newline from `deis logs`

### ISSUE THREAD

**Issue by bacongobbler** [deis/workflow-cli#50]
`deis logs` always prints an extra newline character. We should strip it out.

```
><> deis logs
INFO [go]: domain go removed
INFO [go]: bacongobbler created initial release
INFO [go]: domain go added

```

**Comment by bacongobbler** [comment]
related: https://github.com/deis/workflow/issues/202

**Comment by Joshua-Anderson** [comment]
@bacongobbler This looks like it was fixed in #60, can this be closed?

**Comment by bacongobbler** [comment]
That was a different error. We are still printing an extra newline from `deis logs`

**Comment by Joshua-Anderson** [comment]
Fixed in #165
